pages/profile.tsx

```diff
@@ -73,32 +84,50 @@ const Profile: NextPage = () => {
             <h5 className="text-start fw-semibold mt-5 mb-4">
               Your board colors:
             </h5>
-            <h6 className="text-start">Light:</h6>
-            <input
-              defaultValue={lightColor}
-              {...register("boardLightColor")}
-              type="text"
-              className="form-control mt-3"
-              placeholder={lightColor}
-            />
-            <h6 className="text-start mt-4">Dark:</h6>
-            <input
-              defaultValue={darkColor}
-              {...register("boardDarkColor")}
-              type="text"
-              className="form-control mt-3 mb-5"
-              placeholder={darkColor}
-            />
+            <div className="row mb-5">
+              <div className="col d-flex flex-col align-items-center">
+                <h6 className="text-start">Light:</h6>
+                <SketchPicker
+                  width="70%"
+                  disableAlpha={true}
+                  presetColors={[
+                    "white",
+                    "yellow",
+                    "orange",
+                    "lightblue",
+                    "lightgreen",
+                    "lightpink",
+                  ]}
+                  className="mt-3"
+                  color={boardLightColor}
+                  onChangeComplete={handleLightColorChange}
+                />
+              </div>
+              <div className="col d-flex flex-col align-items-center">
+                <h6 className="text-start">Dark:</h6>
+                <SketchPicker
+                  width="70%"
+                  disableAlpha={true}
+                  presetColors={[
```

Comment:
feel free to change preset colors to nicer light/dark colors i just put the default ones and some random light/dark ones